Cargo.toml

```diff
@@ -35,7 +35,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 [dependencies]
 bech32 = { version = "0.8.1", default-features = false }
 bitcoin_hashes = { version = "0.11.0", default-features = false }
-secp256k1 = { version = "0.24.0", default-features = false }
+secp256k1 = { version = "0.24.0", features = ["bitcoin_hashes"] }
```

Comment:
`default-features` still needs to be `false` to allow `no_std`

Reply:
Oh! Ok! Thanks!

Reply:
Done.